packages/tsx-editor/src/demo/App.tsx

```diff
@@ -1,49 +1,37 @@
 import React from 'react';
 import { PrimaryButton, Stack, Label, mergeStyleSets } from 'office-ui-fabric-react';
-// import { ITranspileOutput } from '../transpiler/transpile.types';
+import { ITextModel } from '../components/Editor.types';
+
+interface ITranspiledOutput {
+  outputString?: string;
+  error?: string;
+}
 
 const classNames = mergeStyleSets({
-  code: {
-    fontFamily: 'monospace',
-    fontSize: 13,
-    lineHeight: '1.5'
-  },
-  renderSection: {
-    backgroundColor: 'red'
-  },
   error: {
     backgroundColor: '#FEF0F0',
     color: '#FF5E79'
   },
-  editor: {
-    width: 800,
-    height: 500
+  component: {
+    backgroundColor: 'lightgray'
   }
 });
 
 interface IAppState {
-  code: string;
-  JScode: string;
   error?: string;
-  fontSize?: string;
   editorHidden?: boolean;
-  editor?: JSX.Element;
-  currentTime: number;
+  editor?: HTMLElement;
 }
 
 export class App extends React.Component {
   public state: IAppState = {
-    code: '',
-    JScode: '',
-    editor: undefined,
-    currentTime: 0,
     editorHidden: true
   };
 
   public render() {
     const editor = (
-      <Stack style={{ backgroundColor: 'lightgray' }} gap={4}>
-        {this.state.editor}
+      <Stack className={classNames.component} gap={4}>
+        {!this.state.editorHidden && this.state.editor}
         {this.state.error !== undefined && <Label className={classNames.error}>`{this.state.error}`</Label>}
```

Comment:
Are there supposed to be backticks in this line? Guessing it should be just:
```suggestion
        {this.state.error !== undefined && <Label className={classNames.error}>{this.state.error}</Label>}
```